spec/Format/L.TileLayer.WMS.Format.js

```diff
@@ -40,4 +73,172 @@ describe('L.TileLayer.WMS.Format', function () {
+      expect(done.calledOnce).to.be.equal(true);
+      expect(L.TileLayer.WMS.Util.AJAX.calledOnce).to.be.equal(true);
+
+      // Check retrieved elements.
+      expect(availableFormats).to.be.an('array');
+      expect(availableFormats).to.have.lengthOf(3);
+      expect(availableFormats).to.have.deep.property('[0]', 'format1');
+      expect(availableFormats).to.have.deep.property('[1]', 'format2');
+      expect(availableFormats).to.have.deep.property('[2]', 'format3');
+
+      // Restore stubbed AJAX util.
+      L.TileLayer.WMS.Util.AJAX = originaAJAX;
+    });
+
+    it('returns exception through \'fail\' callback if AJAX request succeed with text containing \'ServiceExceptionReport\'', function () {
+      var originaAJAX = L.TileLayer.WMS.Util.AJAX;
+
+      L.TileLayer.WMS.Util.AJAX = sinon.spy(function (options) {
+        var responseText = '' +
+          '<?xmlversion="1.0"?>' +
+          '<ServiceExceptionReport version="1.2.0">' +
+          '  <ServiceException code="999" locator="INSERTSTMT01">' +
+          '    parse error: missing closing tag for element WKB_GEOM' +
+          '  </ServiceException>' +
+          '</ServiceExceptionReport>';
+        return options.done(responseText);
+      });
+
+      var fail = sinon.spy(function (xhr) {});
+
+      // Call to 'getAvailable'.
+      L.TileLayer.WMS.Format.getAvailable({
+        url: 'http://test.ru',
+        fail: fail
+      });
+
+      expect(fail.calledOnce).to.be.equal(true);
+      expect(L.TileLayer.WMS.Util.AJAX.calledOnce).to.be.equal(true);
+      var str = fail.getCall(0).args[0].toString();
+      expect(str.indexOf("Error: Unable to parse specified 'xmlString' it isn't valid")).to.be.deep.equal(0);
+
+      L.TileLayer.WMS.Util.AJAX = originaAJAX;
+    });
+
+    it('returns exception through \'fail\' callback if AJAX request succeed with text containing \'ows:ExceptionReport\'', function () {
+      var originaAJAX = L.TileLayer.WMS.Util.AJAX;
+
+      L.TileLayer.WMS.Util.AJAX = sinon.spy(function (options) {
+        var responseText = '' +
+          '<?xmlversion="1.0"?>' +
+          '<ows:ExceptionReport version="1.3.0">' +
+          '  <ows:Exception exceptionCode="XML getFeature request SAX parsing error" locator="test">' +
+          '    <ows:ExceptionText>TEST text' +
+          '    </ows:ExceptionText>' +
+          '  </ows:Exception>' +
+          '</ows:ExceptionReport>';
+        return options.done(responseText);
+      });
+
+      var fail = sinon.spy(function (xhr) {});
+
+      // Call to 'getAvailable'.
+      L.TileLayer.WMS.Format.getAvailable({
+        url: 'http://test.ru',
+        fail: fail
+      });
+
+      expect(fail.calledOnce).to.be.equal(true);
+      expect(L.TileLayer.WMS.Util.AJAX.calledOnce).to.be.equal(true);
+      var str = fail.getCall(0).args[0].toString();
+      expect(str.indexOf("Error: Unable to parse specified 'xmlString' it isn't valid")).to.be.deep.equal(0);
```

Comment:
Same situation here.
Message "Error: XML getFeature request SAX parsing error - TEST text " from the specified ows:ExceptionReport must be expected here, "Error: Unable to parse specified 'xmlString' it isn't valid" indicates that test responseText isn't a valid here (DOMParser can't parse it because in <?xmlversion="1.0"?> space is missing, it must be <?xml version="1.0"?>).
Also 'ows' namsecpace isn't specified in the root <ows:ExceptionReport> element, it must be defined like that '<ows:ExceptionReport version="1.3.0" xmlns:ows="http://www.opengis.net/ows">' (GeoServer and other real GIS Engines always include namespaces in resulting XML documents).
Is it real exception report or just synthetic one?

Also L.TileLayer.WMS.Util.XML.ExceptionReport.OWS parser works not as expected.
It tries to find out <Exception ...> elements inside report, but there are only <ows:Exception> elements,
parser doesn't even try to find them, so the report stays unparsed and final error message contains only 'Error' instead of expected 'Error: XML getFeature request SAX parsing error - TEST text'.

Please, fix L.TileLayer.WMS.Util.XML.ExceptionReport.OWS parser with respect to official schema: [http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd](http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd).